contrib/ansible/roles/kubernetes-addons/templates/skydns-rc.yaml.j2

```diff
@@ -0,0 +1,66 @@
+apiVersion: v1beta3
```

Comment:
I want to run this by you before i do it. I made a replace filter to replace all the pillar vars with the same name var in Ansible. 

So the files will stay up to date but if variable names change we would have to update them.

```
- name: DNS |Download kube-addons.sh file from Kubernetes repo
  get_url: 
    url=https://raw.githubusercontent.com/GoogleCloudPlatform/kubernetes/master/cluster/addons/dns/skydns-svc.yaml.in
    dest=/etc/kubernetes/addons/dns/skydns-svc.yaml
    mode=0755 
    owner=root
    group=root
  tags: dns

- name: DNS | Convert pillar vars to ansible vars for skydns-svc.yaml
  replace: 
    dest=/etc/kubernetes/addons/dns/skydns-rc.yaml
    regexp="pillar\[\'(\w*)\'\]"
    replace="\1"
  tags: dns
```

Let me know if I am too crazy on this one.